src/features/sendEmail/SendEmail.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  Alert,
+  Box,
+  Button,
+  Snackbar,
+  TextField,
+  TextareaAutosize,
+  Typography,
+} from '@mui/material';
+import './SendEmail.css';
+import { useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { useSendEmailMutation } from '../../app/appSlice';
+
+export default function SendEmail() {
+  const { receiverAddress, protectedDataAddress } = useParams();
+
+  //RTK Mutation hook
+  const [sendEmail, result] = useSendEmailMutation();
+
+  //for textarea
+  const [value, setValue] = useState('');
```

Comment:
rename `value` to something more meaningful (ex: `message`)

Reply:
Commit: f2d0e679696c52fad4cf991ee9e7a16eb380884b